llvm/include/llvm/Object/ELF.h

```diff
@@ -321,6 +321,11 @@ class ELFFile {
 
   std::vector<Elf_Rel> decode_relrs(Elf_Relr_Range relrs) const;
 
+  uint64_t crelHeader(ArrayRef<uint8_t> Content) const;
+  using RelsOrRelas = std::pair<std::vector<Elf_Rel>, std::vector<Elf_Rela>>;
```

Comment:
Yes, `std::pair<` instead of an exclusive or is for convenience.

This is a type alias in a class (ELFFile) used by llvm-readobj. lld/ELF doesn't use this class, and the name collision should be fine.